packages/core/useScroll/index.ts

```diff
@@ -161,6 +161,9 @@ export function useScroll(
   const onScrollEndDebounced = useDebounceFn(onScrollEnd, throttle + idle)
 
   const setArrivedState = (target: HTMLElement | SVGElement | Window | Document | null | undefined) => {
+    if (target === window)
+      target = target.document
```

Comment:
```suggestion
      target = target.document.documentElement
```

Maybe we can merge this check into line 168

Reply:
like that?
window？documentElement ：（document？documentELement:el）